iosApp/iosApp/ViewModels/SettingsViewModel.swift

```diff
@@ -94,6 +94,14 @@ class SettingsViewModel: ObservableObject {
                         setting: .searchRouteResults,
                         value: settings[.searchRouteResults] ?? false
                     ),
+                    .toggle(
+                        label: NSLocalizedString(
+                            "Trip Headsigns",
+                            comment: "A setting on the More page to use the destination/headsign on each individual trip in Nearby Transit (only visible for developers)"
```

Comment:
Strictly speaking, `verbatim:` is a SwiftUI `Text` argument and not something `MoreItem.toggle` can do, but since `MoreItem.toggle` takes just `label: String` it's absolutely possible to just pass a literal string directly. (If that weren't the case, it'd also be possible to define this string as don't translate in the `Localizable.xcstrings` editor; I thought about it but decided not to start setting that for this feature flag since we already translated other feature flags.)